php/class-plugin.php

```diff
@@ -99,8 +100,9 @@ public function __construct( $info ) {
 		/**
 		 * Integrations
 		 */
-		$this->components['aws_sqs'] = new AWS_SQS( $this );
-		$this->components['aws_s3']  = new AWS_S3( $this );
+		$this->components['aws_sqs']           = new AWS_SQS( $this );
+		$this->components['aws_s3']            = new AWS_S3( $this );
+		$this->components['gcp_cloud_storage'] = new GCP_Storage( $this );
```

Comment:
Shouldn't this be `storage_gcp` like we discussed the other day? Other classes will be `storage_s3` and `storage_local`.